api/src/repositories/survey-repository.ts

```diff
@@ -244,6 +253,18 @@ export class SurveyRepository extends BaseRepository {
     return new GetSurveyFundingSources(result);
   }
 
+  async getSurveyProprietorDataForSecurityRequest(surveyId: number): Promise<ISurveyProprietorModel> {
```

Comment:
Nit... Want to add JSDoc for the new function

Reply:
done and then some